src/main/resources/sql/maxwell_schema_bootstrap.sql

```diff
@@ -8,6 +8,6 @@ CREATE TABLE IF NOT EXISTS `bootstrap` (
   started_at      DATETIME default NULL,
   completed_at    DATETIME default NULL,
   binlog_file varchar(255) default NULL,
-  binlog_position int unsigned default 0
-);
-
+  binlog_position int unsigned default 0,
+  KEY `is_complete` (`is_complete`,`started_at`)
```

Comment:
don't slip stuff like this into your PR.  it's fine to add an index, but do it as part of another logical PR.